docs/walk_database.md

```diff
@@ -184,8 +184,7 @@ class Command(BaseCommand):
         host = settings.DATABASES['default']['HOST']
 
         con = None
-        con = connect(dbname='postgres', user=user, host = host, password=password)
```

Comment:
This has to be 'postgres' in order for this function to create a database named in the variable 'dbname.'  Subsequent connections would indeed be to 'dbname' but for this first one, it must be 'postgres'

Reply:
Hi @edgarroman,

dbname is name of one database inside of databases kinds. (Kind of databases = Mysql, Postgres, SqlServer). Ex: I have a three databases inside of postgree. FootballScore, RealState and Health. I will connect to postgres but which the database? For this the first param, name of database.

Following your code the dbname should be zappadbname.

PS: the  connect is from psycopg2 so this will only conect to postgres databases.

Reply:
My change is OK.